inference-engine/tests_deprecated/behavior/vpu/myriad_tests/vpu_protocol_tests.cpp

```diff
@@ -26,6 +27,33 @@ TEST_P(MyriadProtocolTests, CanInferenceWithProtocol) {
     ASSERT_EQ(statusCode, StatusCode::OK) << resp.msg;
 }
 
+
+
+TEST_P(MyriadProtocolTests, NoErrorsMessagesWhenLoadNetworkSuccessful) {
+    if (protocol != NC_USB) {
+        GTEST_SKIP();
+    }
+
+    char buff[8192] = {};
+    setbuf(stdout, buff);
+
+    auto network = ie->ReadNetwork(FuncTestUtils::TestModel::convReluNormPoolFcModelFP16.model_xml_str,
+                                   FuncTestUtils::TestModel::convReluNormPoolFcModelFP16.weights_blob);
+
+    std::map<std::string, std::string> config = {{CONFIG_KEY(LOG_LEVEL), CONFIG_VALUE(LOG_WARNING)}};
+
+    InferenceEngine::IExecutableNetwork::Ptr exe_network =
+            ie->LoadNetwork(network, "MYRIAD", config);
+    setbuf(stdout, NULL);
+
+
+    std::string content(buff);
+    for (int i = MVLOG_WARN; i < MVLOG_LAST; i++) {
```

Comment:
Looks very unreliable. And why iterations start from MVLOG_WARN, but not from MVLOG_DEBUG?

Reply:
>And why iterations start from MVLOG_WARN, but not from MVLOG_DEBUG?

because `std::map<std::string, std::string> config = {{CONFIG_KEY(LOG_LEVEL), CONFIG_VALUE(LOG_WARNING)}};`

We expect that when the device is opened successfully, there are no error messages from mvnc. These messages are printed via mvLog. What could go wrong?) Do you have any suggestions?

Reply:
Ok, question just for understanding. I have no suggestion.